docs/opensearch/rest-api/create-indices.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: default
+title: Create indices
+parent: REST API reference
+grand_parent: OpenSearch
+nav_order: 5
+---
+
+# Create indices
+
+While you can create an index by using a document as a base, you can also just create an empty index for use later.
+
+## Example
+
+**Sample Request**
+```json
+PUT /sample-index1
+```
+
+**Sample Response**
+```json
+{
+    "acknowledged": true,
+    "shards_acknowledged": true,
+    "index": "sample-index1"
+}
+```
+
+## Path and HTTP methods
+```
+PUT /<index-name>
+```
+
+## Index naming restrictions
+
+OpenSearch Indices have the following naming restrictions:
+
+- All letters must be lowercase.
+- Index names can't begin with underscores (`_`) or hyphens (`-`).
+- Index names can't contain spaces, commas, or the following characters:
+
+  `:`, `"`, `*`, `+`, `/`, `\`, `|`, `?`, `#`, `>`, or `<`
+
+## URL parameters
+
+You can include the following URL parameters in your request. All parameters are optional.
+
+Parameter | Type | Description
+:--- | :--- | :---
+include_type_name | Boolean | If `true`, the request expects a type in the body of mappings. Default is `false`.
+wait_for_active_shards | String | Specifies the number of active shards that must be available before OpenSearch processes the request. Default is 1 (only the primary shard). Set to `all` or a positive integer. Values greater than 1 require replicas. For example, if you specify a value of 3, the index must have two replicas distributed across two additional nodes for the request to succeed.
+master_timeout | Time | How long to wait for a connection to the master node. Default `30s`.
+timeout | Time | How long to wait for the request to return. Default `30s`.
+
+## Request body
+
+You can also supply the following optional parameters in your HTTP request body.
+
+Parameter | Description
+:--- | :---
+settings | Configuration settings for the index. Available options include `number_of_shards`, `number_of_replicas`, `refresh_interval`, and others.
```

Comment:
We want the full list here unless they belong on another page. So settings.number_of_shards, settings.number_of_replicas, etc., each one an individual row.